sftp/1.4/modules/ROOT/pages/sftp-connector-troubleshooting.adoc

```diff
@@ -0,0 +1,99 @@
+= Troubleshoot SFTP Connector - Mule 4
+
+To troubleshoot Anypoint Connector for SFTP (SFTP Connector), become familiar with the information about troubleshooting connectivity, permissions and server-side issues, troubleshooting Mule application issues, and interpreting commonly thrown messages.
+
+== Troubleshoot Connectivity, Permissions and Server-Side Issues
+
+Sometimes when attempting to connect to an SFTP server you can experience problems establishing the connection because an SFTP requires a configured encrypted communication channel.
+For example, you might need to enable specific encryption cyphers or key exchange algorithms, or indicate the location of certain authorization keys.
+
+To resolve this issue:
+
+. Locate the SFTP servers configuration file to review the parameters and settings regarding the connection, encryption methods, timeouts and so forth, and correct configuration as necessary and customize the environment to suit your needs.
+For example, if you are using OpenSSH you can access the `sshd_config` file.
+
+. Use the configuration file to increase the logging level on the server side.
+
+== Troubleshoot Mule Application Issues
+
+Sometimes you are not sure why the application is failing and you require a better understanding of what is going on in the app's interaction with the SFTP server.
+
+To resolve this error:
+
+* Use the configuration file to increase the logging level on the server side.
+
+* Try to replicate the issue using an SFTP client, for example, Cyberduck or Filezilla. If the issue also occurs with these clients, then it is likely that the issue is not related to the Mule application.
+
+* Get better and more detailed logs on the Mule application side by enabling verbose logging for SFTP Connector and for JSCH driver:
++
+. Access Anypoint Studio and navigate to the *Package Explorer* view.
+. Open your application's project name.
+. Open the *src/main/resources* path folder.
```

Comment:
Not sure if this and the file name should be backticks instead of asterisks. Something to bring up at a future doc meeting for cases like this.